modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc

```diff
@@ -154,4 +159,85 @@ spec:
     secret: binding-request-72ddc0c540ab3a290e138726940591debf14c581 <1>
 ...
 ----
-<1> Unique name of the `Secret` resource generated by the {servicebinding-title}.
\ No newline at end of file
+<1> Unique name of the `Secret` resource generated by the {servicebinding-title}.
+
+[id="workload-resource-mapping_{context}"]
+== Workload resource mapping
+
+[NOTE]
+====
+* Workload resource mapping is available for the secondary workloads of the ServiceBinding custom resource (CR) for both the API groups: `binding.operators.coreos.com` and `servicebinding.io`.
+* You must namespace `ClusterWorkloadResourceMapping` resources only under the `servicebinding.io` API group. However, the `ClusterWorkloadResourceMapping` resources interact with `ServiceBinding` resources under both the `binding.operators.coreos.com` and `servicebinding.io` API groups.
+====
+
+The specification also provides a way to define exactly where binding data needs to be projected. Use this method when you are not able to configure custom path locations correctly by previous methods. You can define `ClusterWorkloadResourceMapping` resources only in the `servicebinding.io` API group, which specify where binding data is to be projected for a given workload kind.
```

Comment:
The following suggestion embodies the following 
- Focus on the user perspective.
- Simplify "you are not able to" to "you cannot"
- Start with the precondition; "If you cannot..."
- Tell the user where to perform the task instead of stating that the task can "only" be performed in a specific location. 

How about:
```suggestion
If you cannot configure custom path locations by using the previous methods, you can define exactly where binding data needs to be projected. Specify where to project the binding data for a given workload kind by defining the `ClusterWorkloadResourceMapping` resources in the `servicebinding.io` API group.
```
- Can we make "previous methods" more specific?

Reply:
If you cannot configure custom path locations by using the configuration method for container path, you can define exactly where binding data needs to be projected. Specify where to project the binding data for a given workload kind by defining the `ClusterWorkloadResourceMapping` resources in the `servicebinding.io` API group.